src/appengine/private/components/upload-testcase/upload-form-simplified.html

```diff
@@ -146,11 +146,11 @@
         </template>
       </div>
       <div class="warning">
-        <b>WARNING</b>: ClusterFuzz doesn't support untrusted workloads.<br>
-        Only upload testcases that you trust enough to run on your own
-        machine. Note that the following jobs do use the Chrome sandbox
-        but this is not a guarantee of protection:<br>
-        [[formatSandboxedJobs(fieldValues.sandboxedJobs)]]
+	      <span><strong>WARNING</strong>: ClusterFuzz only partially supports untrusted workloads.<br></span>
+        ClusterFuzz supports untrusted workloads when the platform is Linux <br>
+        If your workload does not use Linux as the platform, Clusterfuzz will treat it as a trusted workload.
+        Only upload the testcase to ClusterFuzz if you trust it enough to run on your own machine.<br>
```

Comment:
Maybe it needs to be clearer that the second sentence is talking about the same testcase (trusted ones).

Reply:
Changed the wording! lmk if you think that sounds better!